app/src/main/java/org/listenbrainz/android/ui/screens/yim23/Yim23FriendsScreen.kt

```diff
@@ -92,7 +92,7 @@ private fun Yim23Friends (viewModel: Yim23ViewModel) {
                     color = MaterialTheme.colorScheme.background
                 , onClick = {
                         try {
-                            uriHandler.openUri("https://beta.listenbrainz.org/user/${followers.value?.data?.followers!![page]}/year-in-music/2023/")
+                            uriHandler.openUri("https://listenbrainz.org/user/${followers.value?.data?.followers!![page]}/year-in-music/2023/")
```

Comment:
@pranavkonidena can you fetch the base url from constants here?

Reply:
Have made the required changes , please take a look whenever u are able! Thanks